FantasmoSDK/Classes/Extensions/Array+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Array+Extension.swift
+//  FantasmoSDK
+//
+//  Created by Yauheni Klishevich on 20.07.2021.
+//
+
+import Foundation
+
+extension Array {
```

Comment:
Let's not use extensions for syntactic sugar. Also, `!` is already a very clear, common, and succinct way to express negation.